src/pages/product-details/product-details.js

```diff
@@ -1,67 +1,55 @@
-import React, { useEffect, useMemo, useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 
 import { Card } from '@material-ui/core';
 import { faDollarSign, faHryvnia } from '@fortawesome/free-solid-svg-icons';
 import { useStyles } from './product-details.styles';
 
-import Comments from './comments';
-import SimilarProducts from './similar-products';
+import ProductImages from './product-images';
+import ProductInfo from './product-info';
 import ProductSizes from './product-sizes';
 import ProductSubmit from './product-submit';
-import ProductFeatures from './product-features/product-features';
-import ProductInfo from './product-info';
-import ProductImages from './product-images';
+import SimilarProducts from './similar-products';
+import Comments from './comments';
+
 import { Loader } from '../../components/loader/loader';
 import {
   clearProductToSend,
-  getFiltredProducts,
   getProduct,
   setCategoryFilter,
   setProductToSend
 } from '../../redux/products/products.actions';
 
-import { DEFAULT_PRICE, DEFAULT_SIZE } from '../../configs';
 import { selectCurrencyProductsCategoryFilter } from '../../redux/selectors/multiple.selectors';
 
 const ProductDetails = ({ match }) => {
   const { id } = match.params;
-  const {
-    product,
-    isLoading,
-    categoryFilter,
-    productToSend,
-    currency
-  } = useSelector(selectCurrencyProductsCategoryFilter);
+  const { isLoading, productToSend, currency, product } = useSelector(
+    selectCurrencyProductsCategoryFilter
+  );
   const dispatch = useDispatch();
   const styles = useStyles();
 
   const [sizeIsNotSelectedError, setSizeIsNotSelectedError] = useState(false);
 
-  const currencySign =
-    currency === 0 ? faHryvnia : currency === 1 ? faDollarSign : '';
+  let currencySign;
+
+  switch (currency) {
```

Comment:
same

Reply:
CI said, that ternary operator in ternary operator, is code smell, that why I use "switch".

Reply:
you should use ternary and move to utils!